renovate.json

```diff
@@ -2,6 +2,7 @@
     "extends": [
       "config:base"
     ],
+    "baseBranches": [ "master", "1.9" ],
```

Comment:
@ylavoie thanks! We probably need to add this location to the release wiki page as I to add a new branch for a minor/major release, as well as to the list of locations to update when a release goes EOL.

Reply:
Wiki updated